include/library/spdm_common_lib.h

```diff
@@ -918,7 +918,7 @@ libspdm_get_secured_message_context_via_session_info(void *spdm_session_info);
  * @return session info associated with this new session ID.
  **/
 void *libspdm_assign_session_id(void *spdm_context, uint32_t session_id,
-                                bool use_psk);
+                                void *use_psk);
```

Comment:
we cannot change common_lib.h

Reply:
move the change to internal/libspdm_common_lib.h